manifests/config.pp

```diff
@@ -49,6 +49,7 @@
 
   create_resources(keepalived::vrrp::instance, $::keepalived::vrrp_instance)
   create_resources(keepalived::vrrp::script, $::keepalived::vrrp_script)
+  create_resources(keepalived::vrrp::vrrp_track_process, $::keepalived::vrrp_track_process)
```

Comment:
old fashioned method :-D

Are you able to use `.each` instead ?

Reply:
@Dan33l I'll open a separate PR removing all the other `create_resources`.  This should clarify the preferred style.

Reply:
This was merged in https://github.com/voxpupuli/puppet-keepalived/pull/165